raiden_contracts/utils/join-contracts.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+import json
+import os
+import re
+import sys
+
+import click
+from click.types import File
+
+IMPORT_RE = re.compile(r'^import +["\'](?P<contract>[^"\']+.sol)["\'];$')
+
+"""
+Utility to join solidity contracts into a single output file by recursively
+resolving imports.
+
+example usage:
+
+$ cd raiden/smart_contracts
+$ python ../../tools/join-contracts.py SomeContractWithImports.sol joined.sol
```

Comment:
Can you update this comment to use `utils` instead of `tools`?

Reply:
Yes, will do.